Applications/CLI/Tests.cmake

```diff
@@ -191,6 +191,32 @@ if(NOT OGS_USE_MPI)
         tes_zeolite_discharge_large_pcs_0_ts_28_t_1_000000.vtu tes_zeolite_discharge_large_pcs_0_ts_28_t_1.000000.vtu solid_density solid_density
     )
 
+   AddTest(
+        NAME 1D_HeatTransport_dirichlet
+        PATH Parabolic/T/1D_dirichlet
+        EXECUTABLE ogs
+        EXECUTABLE_ARGS line_60_heat.prj
+        WRAPPER time
+        TESTER vtkdiff
+        ABSTOL 1e-2 RELTOL 1e-2
```

Comment:
I'd say that in absence of any singularities in the solution, the tolerances are a little to sloppy. Why is it so? Can we improve it?